docs/metadata.rst

```diff
@@ -131,23 +131,28 @@ encoding is not easy to find, its length is added in a two-byte big-endian
 encoding. The current version of the Solidity compiler usually adds the following
 to the end of the deployed bytecode::
 
-    0xa1 0x65 'b' 'z' 'z' 'r' '0' 0x58 0x20 <32 bytes swarm hash> 0x00 0x29
+    0xa2
+    0x65 'b' 'z' 'z' 'r' '0' 0x58 0x20 <32 bytes swarm hash>
+    0x64 's' 'o' 'l' 'c' 0x43 <3 byte version encoding>
```

Comment:
I think the version should already be mentioned in the first paragraph.

Reply:
Updated.